data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+	to offer
+		has "Hai Rescue [C05] Operatives: done"
+	on visit
+		dialog `You forgot some of the operatives or their equipment; they're probably in your escorts. Wait for the escorts to arrive and land again to finish this mission.`
+	on offer
+		log `Agent Kopkow of Republic Intelligence complained that they're running out of cover stories to hide the Hai. He needs the Hai-human conflict resolved without an more public displays. Once I have space for his 200 operatives, I need to pick them up at the Foundry spaceport.`
+		conversation
+			`Soon after landing, you are met by Agent Kopkow, the Republic Intelligence agent you met on Earth. He seems extremely concerned about something.`
+			`	"It is becoming increasingly difficult to cover up the existence of the Hai. The captains at Ultima Thule already know about the Hai, but we had to tell the crews we were dealing with a major pirate threat. We hired a few pirate gangs, tricking them into flying through the area to back our story. The Shield Beetles at Prime were explained away as experimental ship designs."`
+			`	"Our cover stories have passed the point of plausibility into the realm of barefaced lying. I hope you can finish this without too many more public displays, otherwise we won't be able to hide the Hai anymore.`
+			`	"I've brought 200 intelligence operatives, specially trained for attacking pirate strongholds, and equipped with advanced weapons that... are on a need-to-know basis. Unfortunately, we possess no ships that can transport that many people safely to the surface. Even a Bactrian would be torn to pieces. You will have to transport the operatives. We can transport them back afterwards."`
+			`	"Once you're ready for the battle at Philosopher's Paradise, meet our operatives in the spaceport. You will need 200 bunks and 70 tons of cargo space to fit all the operatives and their gear. Please don't take long. We're running out of cover stories."`
+			`	He walks off dismayed.`
+				accept
+	on complete
+		payment 600000
+
+
+mission "Hai Rescue [C07A] Devil-Hide Fleet"
+	landing
+	invisible
+	source "Foundry"
+	destination "Hai-home"
+	to offer
+		has "Hai Rescue [C06] Meet the Fleet: done"
+# Workaround for bug #1936 is to introduce another bug the user is unlikely to
+# trigger. Visit Hai-home with this fleet and it vanishes.
+#	to complete
+#		has "Hai Reveal [D01] Cleanup: done"
+	to fail
+		or
+			has "Hai Rescue [C07] Devil-Hide Battle: failed"
+			has "Hai Rescue [C07] Devil-Hide Battle: declined"
+			has "Hai Reveal [D01] Cleanup: failed"
+			has "Hai Reveal [D01] Cleanup: declined"
+			has "Hai Rescue [C08] Freedom: failed"
+			has "Hai Rescue [C08] Freedom: declined"
+	npc
+		government "Hai (Friendly Unfettered)"
+		personality escort heroic
+		fleet 5
+			variant
+				"Shield Beetle (Particle Jump)"
+		fleet 5
+			variant
+				"Shield Beetle (Plasma Blaster Jump)"
+
+mission "Hai Rescue [C07] Devil-Hide Battle"
```

Comment:
A question: how does this mission work? It states that the completion criteria is to dominate the planet, but as far as I know there isn't actually a way to make dominating a planet a mission. I'm not code-smart enough to understand how this works after 5 minutes of staring at it.

Reply:
It works in the sense that there are precisely 0 ways to land without dominating the planet (it cannot be bribed) so you cannot reach the destination of the mission without dominating.